src/ray/rpc/server_call.h

```diff
@@ -157,10 +157,11 @@ class ServerCallImpl : public ServerCall {
     // We create this before handling the request so that the it can be populated by
     // the completion queue in the background if a new request comes in.
     factory.CreateCall();
+    auto reply = std::make_shared<Reply>();
```

Comment:
Could you document that we want the reply to be destructed once it's sent instead of being destructed on the next request arriving?